lib/ts/recipe/emailpassword/recipe.ts

```diff
@@ -101,6 +105,183 @@ export default class Recipe extends RecipeModule {
                 Recipe.instance = new Recipe(Recipe.RECIPE_ID, appInfo, isInServerlessEnv, config, {
                     emailDelivery: undefined,
                 });
+
+                PostSuperTokensInitCallbacks.addPostInitCallback(() => {
```

Comment:
this should be in the constructor of the recipe? The combinede recipes like thirdpartyemailpassword don't call the init function.. they directly create an instance of this recipe no? How is this not tested?

Reply:
Done